posydon/visualization/plot_defaults.py

```diff
@@ -404,7 +404,7 @@
             ['o', 2, 'tab:brown', 'stripped He-star C depletion'],
         'stripped_He_non_burning':
             ['o', 2, 'gray', 'stripped He-star non burning'],
-        'stripped_He_Core_H_burning':
+        'accreted_He_Core_H_burning':
             ['x', 1, 'black', 'unknown'],
```

Comment:
Please keep the old entry and add a new one for `accreted_He_Core_H_burning`, which is a known state. (The same applies to the other case, where you replaced `stripped_He_Core_H_burning` in the plot defaults.

Reply:
Didn't we remove the "stripped_He_Core_H_burning" state so it no longer exists? So we can replace this and then yes also edit the label too

Reply:
It is not removed. We replaced it in the cases, where it was wrongly assigned.